ui/common/css/theme/board/_chessground.scss

```diff
@@ -212,3 +212,53 @@ html.transp body:not(.simple-board) cg-board {
     //filter: hue-rotate(calc(var(---board-hue) * 3.6deg));
   }
 }
+
+coords.squares {
```

Comment:
I don't love the `.squares` class name. Squares are referenced very often in chess. For example, chessground already uses a `<square>` element. I anticipate conflicts one day, but for now, we can just specify the `<coords>` element to mitigate that risk.